src/main/java/org/zalando/nakadi/domain/EventType.java

```diff
@@ -74,6 +80,8 @@ public EventType(final String name, final String topic, final String owningAppli
         this.schema = schema;
         this.defaultStatistic = defaultStatistic;
         this.options = options;
+        this.writeScope = writeScope;
+        this.readScope = readScope;
```

Comment:
why the names are singular?
Those are sets, I think the names should be plural.